### ISSUE TITLE
wily graph gives "plotly.exceptions.PlotlyEmptyDataError: Empty data list found. Make sure that you populated the list of data objects you're sending and try again."

### ISSUE THREAD

**Issue by pzelnip** [tonybaloney/wily#17]
I've run wily on a project here at work, and while a `wily report ....` works for every file I try, any time I do a `wily graph ...` with any file I get:

```
Traceback (most recent call last):
  File "/Users/pzelnip/.virtualenvs/myproj/bin/wily", line 11, in <module>
    sys.exit(cli())
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/wily/__main__.py", line 224, in graph
    graph(config=config, paths=[files], metrics=metrics, output=output)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/wily/commands/graph.py", line 92, in graph
    filename=filename,
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/plotly/offline/offline.py", line 618, in plot
    '100%', '100%', global_requirejs=False)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/plotly/offline/offline.py", line 244, in _plot_html
    figure = tools.return_figure_from_figure_or_data(figure_or_data, validate)
  File "/Users/pzelnip/.virtualenvs/myproj/lib/python3.6/site-packages/plotly/tools.py", line 1521, in return_figure_from_figure_or_data
    "Empty data list found. Make sure that you populated the "
plotly.exceptions.PlotlyEmptyDataError: Empty data list found. Make sure that you populated the list of data objects you're sending and try again.
Questions? Visit support.plot.ly
```

Is there an extra setup step to getting graph to work?  All I've done is the `pip install wily` since that's all that's mentioned in the [docs](https://wily.readthedocs.io/en/latest/)

In case it matters:

Python 3.6.2
OS: OSX 10.12.6 (Sierra)
wily==1.4.0

**Comment by pzelnip** [comment]
Nevermind, I'm dumb, I forgot to specify the metric to use:

```
wily graph path/to/my/file maintainability.mi
```
I don't know if it's worth keeping this issue open to improve the error message you get when you don't specify a metric, will leave it up to you if this should be left open or closed.

**Comment by tonybaloney** [comment]
that should definitely be resolved. It should use the default metrics if none are specified, however that's clearly not working

**Comment by tonybaloney** [comment]
Fixed in 1.5.0.dev0

**Comment by tonybaloney** [comment]
released in 1.5.0
